dynawo/sources/API/PAR/PARReference.cpp

```diff
@@ -27,7 +27,14 @@ using std::string;
 
 namespace parameters {
 
-Reference::Reference(const string& name) : type_(""), name_(name), origName_(""), componentId_("") {}
+Reference::Reference(const string& name, OriginData origData) :
+  type_(""),
```

Comment:
useless (same for all strings)

Reply:
All these attributes can be accessed in the code. It may be dangerous to not initialize them all.

Reply:
The default constructor of strings create an empty string so it is useless to use the copy constructor for an empty string